modules/distr-tracing-tempo-troubleshoot-collecting-diagnostic-data-from-command-line.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="collecting-diagnostic-data-from-command-line_{context}"]
```

Comment:
This is the same exact ID as the other module. Recommend making them slightly different

Reply:
OK. Tweaked both.